src/test/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundEdifactTransactionHandlerTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.nhs.digital.nhsconnect.lab.results.inbound;
+
+import org.hl7.fhir.dstu3.model.Parameters;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.nhs.digital.nhsconnect.lab.results.inbound.fhir.EdifactToFhirService;
+import uk.nhs.digital.nhsconnect.lab.results.inbound.queue.DataToSend;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Transaction;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class InboundEdifactTransactionHandlerTest {
```

Comment:
Are there any other cases we can test here? A single test always looks sus to me, but could be legitimate.

Reply:
I've decided to remove `InboundEdifactTransactionHandler.java` for simplicity - for now, as there wasn't a lot going on in there!